Fixes #3641 Final Serving Sanity Check (#3747)
knative/docs: docs/serving/using-a-custom-domain.md

1. Edit the domain configuration config-map to replace `example.com` with your
   own domain, for example `mydomain.com`:

     ```bash
     kubectl edit configmap config-domain -n knative-serving
     ```

>      This command opens your default text editor and allows you to edit the [config
>      map](https://github.com/knative/serving/blob/main/config/core/configmaps/domain.yaml).

Reviewer: ```suggestion
     This command opens your default text editor and allows you to edit the [ConfigMap](https://github.com/knative/serving/blob/main/config/core/configmaps/domain.yaml).
```

1. Edit the domain configuration config-map to replace `example.com` with your
   own domain, for example `mydomain.com`:

     ```bash

>      kubectl edit cm config-domain --namespace knative-serving

Reviewer: cm -> configmap
Use the style guide for these

>      kubectl edit cm config-domain --namespace knative-serving

Reviewer: ```suggestion
     kubectl edit configmap config-domain -n knative-serving
```

     ```

     This command opens your default text editor and allows you to edit the [config
     map](https://github.com/knative/serving/blob/main/config/core/configmaps/domain.yaml).

     ```yaml
     apiVersion: v1
     data:
       _example: |
         ################################
         #                              #
         #    EXAMPLE CONFIGURATION     #
         #                              #
         ################################
         # ...
         example.com: |
     kind: ConfigMap
     ```

> 1. Edit the file to replace `example.com` with the domain you'd like to use, remove the `_example` key and save your changes. In this example, we configure `mydomain.com` for all routes:

Reviewer: ```suggestion
1. Edit the file to replace `example.com` with the domain you want to use, then remove the `_example` key and save your changes. In this example, `mydomain.com` is configured as the domain for all routes:
```

     ```yaml
     apiVersion: v1
     data:
       mydomain.com: ""
     kind: ConfigMap
     [...]
     ```

## Apply from a file

You can also apply an updated domain configuration:

> 1. Create a new file, `config-domain.yaml` and paste the following text, replacing the `example.org` and `example.com` values with the new domain you want to use:

Reviewer: use the kubectl apply command from the style guide instead of saying copy/paste into a yaml file

     ```yaml
     apiVersion: v1
     kind: ConfigMap
     metadata:
       name: config-domain
       namespace: knative-serving
     data:
       # These are example settings of domain.
       # example.org will be used for routes having app=prod.
       example.org: |
         selector:
           app: prod
       # Default value for domain, for routes that does not have app=prod labels.
       # Although it will match all routes, it is the least-specific rule so it
       # will only be used if no other domain matches.
       example.com: ""
     ```

1. Apply updated domain configuration to your cluster:

     ```bash
     kubectl apply --filename config-domain.yaml

>      ```

Reviewer: these apply commands can be removed if you're following the style guide for the kubectl edit commands

## Deploy an application

> > If you have an existing deployment, Knative will reconcile the change made to the configuration map and automatically update the host name for all of the deployed services and routes.

Reviewer: ```suggestion
> If you have an existing deployment, Knative reconciles the change made to the ConfigMap, and automatically updates the host name for all of the deployed Services and Routes.
```

Author: love that active voice
